rs/crypto/internal/crypto_service_provider/src/signer/tests.rs

```diff
@@ -483,27 +394,6 @@ mod verify_ed25519 {
         );
     }
 
-    #[test]
-    fn should_fail_to_verify_batch_under_wrong_public_key() {
```

Comment:
In `ic-crypto-ed25519` we only test if swapping keys causes verification to fail. I'm not sure if this is additional case is that interesting / worth extending the tests for.

Reply:
I extended the tests accordingly. PTAL

Reply:
@randombit, so that I can merge, I'm resolving this open conversation because you approved the MR.